otx/core/action_dataset_adapter.py

```diff
@@ -0,0 +1,169 @@
+    def convert_to_otx_format(self, datumaro_dataset: dict) -> DatasetEntity:
+        label_information = self._prepare_label_information(datumaro_dataset)
+        label_entities = label_information["label_entities"]
+
+        label_schema = self._generate_default_label_schema(label_entities)
+        dataset_items = []
+        for subset, subset_data in datumaro_dataset.items():
+            for datumaro_items in subset_data:
+                for datumaro_item in datumaro_items:
+                    image = Image(file_path=datumaro_item.media.path)
+                    shapes = []
+                    for ann in datumaro_item.annotations:
+                        # Action Classification
+                        if isinstance(ann, DatumaroLabel):
+                            shapes.append(
+                                Annotation(
+                                    Rectangle.generate_full_box(),
+                                    labels = [
+                                        ScoredLabel(
+                                            label=label_entities[ann.label]                                        )
+                                    ]
+                                )
+                            )
+                    # Unlabeled dataset
+                    if len(shapes) == 0:
+                        annotation_scene = NullAnnotationSceneEntity()
+                    else:
+                        annotation_scene = AnnotationSceneEntity(kind=AnnotationSceneKind.ANNOTATION, annotations=shapes)
+                    dataset_item = DatasetItemEntity(image, annotation_scene, subset=subset)
+                    dataset_items.append(dataset_item)
+        return DatasetEntity(items=dataset_items), label_schema
+
+class ActionDetectionDatasetAdapter(ActionBaseDatasetAdapter, BaseDatasetAdapter):
+    def convert_to_otx_format(self, datumaro_dataset: dict) -> DatasetEntity:
+        label_information = self._prepare_label_information(datumaro_dataset)
+        label_entities = label_information["label_entities"]
+
+        label_schema = self._generate_default_label_schema(label_entities)
+        dataset_items = []
+        for subset, subset_data in datumaro_dataset.items():
+            for datumaro_items in subset_data:
+                for datumaro_item in datumaro_items:
+                    image = Image(file_path=datumaro_item.media.path)
+                    shapes = []
+                    for ann in datumaro_item.annotations:
+                        if isinstance(ann, DatumaroBbox):
+                            shapes.append(
+                                Annotation(
+                                    Rectangle(
+                                        x1=ann.points[0]/image.width, 
+                                        y1=ann.points[1]/image.height,
+                                        x2=ann.points[2]/image.width,
+                                        y2=ann.points[3]/image.height),
```

Comment:
do we need to have normalized position information during import?

Reply:
Currently, OTX needs a normalized position.

Reply:
A bit late to the party here, but this is actually a legacy code from NOUS days. Backend team also want to remove the normalized coordinates. It might be an idea to take the iniatative to remove this. 

we normalize denormalize and normalize again during the workflow, which is not efficient at all.